src/css/main.css

```diff
@@ -870,7 +870,7 @@ span.subcomponent {
 }
 
 code, pre {
-  font-family: Consolas, Andale Mono, Monaco, Courier New, monospace;
+  font-family: 'Roboto Mono' Consolas, Andale Mono, Monaco, Courier New, monospace;
```

Comment:
you should listing `'Roboto Mono'` only when WebFont has loaded, like this:

```css
.wf-active code, .wf-active pre {
```

and then keep the others set.